addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -541,6 +541,17 @@ export class Wysiwyg extends Component {
         );
 
         this.$editable.on('click', '.o_image, .media_iframe_video', e => e.preventDefault());
+        this.$editable.on('click', '.o_editor_banner > i, .o_editor_banner > span', event => {
```

Comment:
Would prefer a specific class on the icon.

Currently it's a bit fragile as if we add a div wrapping the icon, it would not work anymore.

Reply:
I disagree on that one, I think a specific class selector to apply custom behavior is always preferable rather than relying on something that will be very easily broken if we modify the XML structure.

> If we ever wrap the icon in a div or something, we will simply have to change the selector here.

I can almost guarantee you that we will not do it :smile: 

> Since the class will be stored in the document and in the database, I'd rather avoid adding a new class if it's not strictly necessary.

I don't think it's a huge deal, it only applies to this specific block, more robust and clear code is worth a few bytes in very specific cases.